cmd/minishift/cmd/config/set.go

```diff
@@ -69,10 +69,12 @@ func set(name string, value string) error {
 		return err
 	}
 
-	// Run any callbacks for this property
-	err = run(name, value, s.callbacks)
-	if err != nil {
-		return err
+	if runCallback {
+		// Run any callbacks for this property
+		err = run(name, value, s.callbacks)
+		if err != nil {
+			return err
+		}
```

Comment:
@praveenkumar I am wondering about the change. Can you please add why this is required?